modules/libffi/3.4.7.bcr.3/MODULE.bazel

```diff
@@ -0,0 +1,7 @@
+module(
+    name = "libffi",
+    version = "3.4.7.bcr.3",
+)
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "platforms", version = "0.0.11")
+bazel_dep(name = "rules_license", version = "1.0.0")
```

Comment:
Shouldn't `rules_cc` be here too?

Reply:
Yep, you are right, it was still using the native ones... I added it.